package.json

```diff
@@ -18,7 +18,7 @@
   "author": "AlaskaAirlines",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=14.17.6"
```

Comment:
Should we raise this to 16 now that the node-sass issue is resolved?

Reply:
We 'can' go to 16... but I feel like we are not really making that decision based on any real data. I'd like to set a policy where there is a clear expectation from our users. See this issue https://github.com/AlaskaAirlines/WC-Generator/issues/218

Given the statement from Semantic Release, we 'could' take the position of supporting the last LTS version, i.e. v16.13. 

That being said, we need to be developing against `current` in preparation for the next release.